core/src/main/java/nu/yona/server/subscriptions/service/BuddyConnectResponseMessageDto.java

```diff
@@ -137,6 +140,7 @@ MessageActionDto handleAction_Process(UserDto actingUser, BuddyConnectResponseMe
 						connectResponseMessageEntity.getRelatedUserAnonymizedId().get(),
 						connectResponseMessageEntity.getSenderNickname());
 			}
+			UserDto actingUserAfterUpdate = userService.getPrivateUser(actingUser.getId());
```

Comment:
The user DTO has changed, and the response of this method should use the up-to-date user DTO. I update it here because the actions are fired here.